man/systemd.network.xml

```diff
@@ -3491,6 +3491,14 @@ Token=prefixstable:2002:da8:1::</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MPUBytes=</varname></term>
+        <listitem>
+          <para>Rounds each packet (including overhead) up to the specified bytes. Takes an unsigned
```

Comment:
I'd drop the "unsigned", given that the range you document 0…256 makes very clear it's unsigned, the info is thus redundant.